modules/ROOT/pages/release-notes.adoc

```diff
@@ -304,6 +306,17 @@ A standalone docker image that package the LDAP Synchronizer was released under
 * RUNTIME-212 - Process manager typo into dynamic-permissions-checks(-custom).properties file
 * RUNTIME-33 - Impossible to install a valid BDM after an attempts to install an invalid BDM
 
+=== Fixes in Bonita 2021.1-0915 (2021-09-15)
+
+==== Fixes in Bonita Development Suite (Studio and UI Designer)
+
+* STUDIO-4109 - LA-Builder requires SP dependency in its classpath
+
+==== Fixes in Bonita Runtime (including Portal)
+
+* RUNTIME-479 -	CVEs in OpenSSL version in Docker images
```

Comment:
We need to be really careful when disclosing CVE fixes to the public.
If it does imply that customers who currently run previous versions of Bonita are vulnerable, we need to stay very generic about the nature of the fixes, so no malicious intention can lead to damages in our customers' Bonita installation(s).